operators/src/processing/temporal_raster_aggregation/subquery.rs

```diff
@@ -122,6 +122,7 @@ where
         Ok(RasterTile2D::new(
             time,
             tile_position,
+            0,
```

Comment:
good point. It doesn't matter actually, because we overwrite the band index in the adapter anyway. At least if we decide to do it like that, so let's about this and then I'll add a comment here or change the code.

Reply:
we can remove that now right?

Reply:
I changed here that the index is only set after the tile is produced. So the accumulator does not need to know the band anymore and just can always use band index 0 053e567c1b019f9f5f364f41e6b08537fea0d2f3